src/api/projects/projects.service.ts

```diff
@@ -71,6 +71,19 @@ export class ProjectsService {
     return project;
   }
 
+  public async findOneBySlug(slug: string, user: User) {
+    const project = await this.prisma.project.findFirst({ where: { slug } });
+    if (!project) throw new NotFoundError('Project not found');
+    if (!user || user.role === Role.USER || project.organizationId !== user.organizationId) {
+      delete project.mpAccessToken;
+      delete project.mpPublicKey;
```

Comment:
I remove the mpPulicKey from the excluded fields, as could be used in the FE. Could you remove it here too?